docs/sql-guide/functions/function-setexcludes.md

```diff
@@ -0,0 +1,58 @@
+---
+title: SETEXLCUDES()
+layout: default
+parent: Functions
+grand_parent: SQL guide
+---
+
+# SETEXCLUDES() function
+
+`SETEXCLUDES()` tests membership of a set of values within a set. It returns true if all of the members of `testset` do not exist in `targetset`
+
+## Syntax
+
+```
+setexcludes(targetset, testmember)
+```
+
+## Arguments
+
+| Argument | Description | Data type |
```

Comment:
An alternative to the "Before you begin" heading suggested above would be to add a fourth column "Additional information".
Into this you can put the IDSET and STRINGSET links from above for the first row.
The second row doesn't have anything really to add so you can just add a column with nothing in it